happi/cli.py

```diff
@@ -64,6 +64,10 @@ def happi_cli(args):
     parser = get_parser()
     args = parser.parse_args(args)
 
+    # print happi usage if no arguments are provided
+    if len(sys.argv) == 1:
+        parser.print_usage()
+
```

Comment:
I think it should probably return here after printing the usage.

Reply:
I added `sys.exit(0)` in 5bbfc78 - Is this what you meant @klauer ? or should i use a different exit code, or an actual return?

Reply:
The problem with `sys.exit` is that it's a heavy-handed solution and doesn't respect the fact that the caller may not want the process to exit. Returning is, in my opinion, most appropriate here.

Reply:
Ok thank you, that sounds good, changed it in 8826d6d

Reply:
i'm failing the build though, let me take a look

Reply:
Strange - try `pytest -vv --pdb` with/without the `-s` and look at the backtrace.

Reply:
The problem for me is that as soon as I add some parameters like `-vv` or `--pdb` I don't get the errors anymore, so can't look at the backtrack - hahah - I don't get it. 
I change the implementation in 4d6cc28 and it works this way... but I still want to know why it was not working the other way.

Reply:
Oh, looks like because `sys.argv` is getting your `pytest` arguments and not `happi` CLI arguments. That'll do it for sure.